tribits/examples/TribitsOldSimpleExampleApp/README.md

```diff
@@ -21,11 +22,21 @@ After building and installing TribitsExampleProject under
   ctest
 ```
 
-This project can be instructed to use the old deprecated targets by adding the
-CMake cache var:
+This project can be instructed to use the non-namespaced old deprecated
+targets by adding the CMake cache var:
 
 ```
   -D TribitsOldSimpleExApp_USE_DEPRECATED_TARGETS=ON
 ```
 
-This will generated deprecated warnings with newer versions of TriBITS.
+This will generated deprecated warnings with newer versions of TriBITS.  (This
+is to demonstrate and test these deprecated non-namespaced targets and to
+demonstrate how to remove the deprecated warnings in a downstream customer
+CMake project.)
+
+NOTE: The modern version of this project using the modern CMake targets
+produced by `TribitsExampleProject` is
```

Comment:
Can be changed by the link :
[`TribitsExampleProject`](../TribitsExampleProject/README.md)

Reply:
Same comment as [above](https://github.com/TriBITSPub/TriBITS/pull/440#discussion_r820942054).